### PR TITLE
Add urls

### PR DESCRIPTION
**Pull request by Towhid1**
## Description of changes

*Summarize the changes made by this PR.*
- add_image_urls added
- example.py updated

### CODE REVIEW

File: python/bagel/api/fastapi.py [hunk 1]
```diff
@@ -86,6 +86,7 @@ def create_cluster(
             id=resp_json["id"],
             name=resp_json["name"],
             metadata=resp_json["metadata"],
+            cluster_size=resp_json["cluster_size"],
```
Comment (BidhanRoy) [inline python/bagel/api/fastapi.py:89]: why do we need cluster size as param?

File: python/bagel/api/fastapi.py [hunk 2]
```diff
@@ -427,6 +429,34 @@ def get_version(self) -> str:
...
+        self,
+        cluster_id: UUID,
+        ids: IDs,
+        urls: List[str],
+        metadatas: Optional[Metadatas] = None,
+        increment_index: bool = True,
+    ) -> Any:
+        """Add image by urls to BagelDB."""
+        if metadatas is None:
+            metadatas = [{"url": str(url)} for url in urls]
```
Comment (BidhanRoy) [inline python/bagel/api/fastapi.py:443]: url will not be added if metadata exists?

Comment (Towhid1) [reply python/bagel/api/fastapi.py:443]: Yes. if there is no metadata then we will add URL as metadata.

File: python/bagel/api/fastapi.py (hunk 1 above)
Comment (Towhid1) [reply python/bagel/api/fastapi.py:89]: To share cluster_size with users.

**Pull request merged**